tests/Fixer/FunctionNotation/VoidReturnFixerTest.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\FunctionNotation;
+
+use PhpCsFixer\Test\AbstractFixerTestCase;
+
+/**
+ * @author   Mark Nielsen
+ *
+ * @internal
+ *
+ * @requires PHP 7.1
+ * @covers   \PhpCsFixer\Fixer\FunctionNotation\VoidReturnFixer
+ */
+final class VoidReturnFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @dataProvider provideFixCases
+     *
+     * @param string      $expected
+     * @param null|string $input
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFixCases()
```

Comment:
👎 For that. It would indicate a programmer error more likely in the function body, particularly if they are implementing an interface with return types.